### PR TITLE
chore(#1233): Document CHT installation in new Mac

### PR DESCRIPTION
**Pull request by Benmuiruri**
# Description

Update [CHT Core dev environment setup](https://docs.communityhealthtoolkit.org/contribute/code/core/dev-environment/#couchdb-setup-in-cht-4x) to include setup instructions for people using Apple Silicon Macbooks.

In the current Docs, the section [CouchDB Setup in CHT 4.x](https://docs.communityhealthtoolkit.org/contribute/code/core/dev-environment/#couchdb-setup-in-cht-4x:~:text=echo%20%24COUCH_URL-,CouchDB%20Setup%20in%20CHT%204.x,-Create%20a%20docker) includes a command that creates a `docker-compose.yml` file that points to the publicly available images at `public.ecr.aws/medic/cht-couchdb:4.4.0-alpha`

Unfortunately, the images available on there are built for `AMD ` architecture

So when you run the command `COUCHDB_USER=medic COUCHDB_PASSWORD=password docker-compose -f docker-compose.yml -f couchdb-override.yml up -d`

You get this error 
`couchdb The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested`

Because of the lack of `ARM` images, the current workaround is to build the correct images locally.

#1233 

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by Benmuiruri** [comment]
Hi @ralfudx  please check whether the steps I have outlined are correct and see whether I missed anything. 

Also, if there is something you experienced that might be good to add in the [Troubleshooting](https://docs.communityhealthtoolkit.org/contribute/code/core/dev-environment/#other-path-troubleshooting)  section, please let me know and I can add. 

Thanks.

File: content/en/contribute/code/core/dev-environment.md
Comment (ralfudx) [inline content/en/contribute/code/core/dev-environment.md:150]: can we also confirm that the folder has three files cht-core.yml, cht-couchdb.yml & cht-couchdb-clustered.yml

File: content/en/contribute/code/core/dev-environment.md
Comment (ralfudx) [inline content/en/contribute/code/core/dev-environment.md:175]: can we try to kill all running containers before starting this one?
`docker` kill $(docker ps -q)`

File: content/en/contribute/code/core/dev-environment.md
Comment (Benmuiruri) [reply content/en/contribute/code/core/dev-environment.md:150]: yes, that's a good suggestion.

**Comment by Benmuiruri** [comment]
Hi @esthermmoturi  please review this documentation for the writing style. Both @ralfudx  and I confirm the steps are correct for someone setting up on Macs with Apple Silicon. 

Thank you

**Review by ralfudx — APPROVED** [review]

**Comment by esthermmoturi** [comment]
Hey @Benmuiruri I'll look through the changes and test.Is this specific for M2 chips or it applies for M1 chips as well? Also could you kindly highlight the issues/errors that informed the change in documentation.Kindly add this to the PR description.

Thanks.

**Comment by Benmuiruri** [comment]
Hi @esthermmoturi , both M1 and M2 use `arm64` architecture. 

In the current Docs, the section [CouchDB Setup in CHT 4.x](https://docs.communityhealthtoolkit.org/contribute/code/core/dev-environment/#couchdb-setup-in-cht-4x:~:text=echo%20%24COUCH_URL-,CouchDB%20Setup%20in%20CHT%204.x,-Create%20a%20docker) includes a command that creates a `docker-compose.yml` file that points to the publicly available images at `public.ecr.aws/medic/cht-couchdb:4.4.0-alpha`

Unfortunately, the images available on there are built for `AMD ` architecture

So when you run the command `COUCHDB_USER=medic COUCHDB_PASSWORD=password docker-compose -f docker-compose.yml -f couchdb-override.yml up -d`

You get this error 
`couchdb The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested`

Because of the lack of `ARM` images, Diana suggested[ building the images locally ](https://forum.communityhealthtoolkit.org/t/cht-docker-image-not-running-on-apple-silicon/3038/5#:~:text=I%20think%20the%20easiest%20way%20to)

**Comment by latin-panda** [comment]
@Benmuiruri, it's been some weeks since the last update in this PR. Is it ready for merge?

**Comment by esthermmoturi** [comment]
Hey @latin-panda , not yet. I got some blockers while testing the documentation , hopefully I'll approve it in a few hours(if everything works as expected).

**Review by esthermmoturi — APPROVED** [review]
Hey @Benmuiruri , I have added some suggestions and will follow up with the issues I experienced separately.

**Comment by Benmuiruri** [comment]
Hi @esthermmoturi I confirmed that the error you were facing is not blocking this PR, I will merge this PR.

**Pull request merged**
